qt/widgets/common/src/MantidTreeWidget.cpp

```diff
@@ -23,9 +23,9 @@ Mantid::Kernel::Logger treelog("MantidTreeWidget");
 namespace MantidQt {
 namespace MantidWidgets {
 
-MantidTreeWidget::MantidTreeWidget(QWorkspaceDockView *w,
-                                   MantidDisplayBase *mui)
-    : QTreeWidget(w), m_dockWidget(w), m_mantidUI(mui),
+MantidTreeWidget::MantidTreeWidget(MantidDisplayBase *mui,
+								   QWidget *parent)
+    : m_mantidUI(mui),
```

Comment:
I think this line should now read: ` QTreeWidget(parent), m_mantidUI(mui),` as I'm getting a compiler warning about not using `parent`.